apps/safe-claiming-app/src/hooks/useDarkMode.ts

```diff
@@ -12,11 +12,8 @@ export const useDarkMode = (): boolean => {
   const [isDarkMode, setIsDarkMode] = useState<boolean>(false)
 
   useEffect(() => {
-    const urlParams = new URLSearchParams(location.search)
-    const urlDarkMode = urlParams.get("theme")
-
-    setIsDarkMode(urlDarkMode === "dark" || isSystemDarkMode())
-  }, [location.search])
+    setIsDarkMode(location.hash.endsWith("+dark") || isSystemDarkMode())
```

Comment:
We shouldn't use `isSystemDarkMode`. First of all, web-core will always send +dark if needed, and second of all, the web-core setting can be different than the system setting.

Reply:
If you want to use the system fallback for the main claiming app, then the widget should explicitly send +light. Otherwise, you'll always fallback to the OS setting which might not be correct.

Reply:
I think we don't need to fallback to the system setting at all. I'm gonna remove it given web-core always explicitly passes dark / light for the theme.

Reply:
I now explicitly check if `+light` is passed. If none of the theme preferences is passed in the URL then we are rendering the main Claiming app which uses the system preference